tests/functional/conftest.py

```diff
@@ -12,6 +12,7 @@
 import gitlab
 import gitlab.base
 from tests.functional import helpers
+from tests.functional.pytest_docker_fixtures import *  # noqa
```

Comment:
One interesting thing is that our `tests/functional/fixtures/` directory contains all the fixture files related to this docker setup. So maybe we could move this to `tests/functional/fixtures/docker.py` if that makes sense? It would probably need an `__init__.py` in that folder. WDYT?

```suggestion
from tests.functional.fixtures.docker import *  # noqa
```

Reply:
Totally makes sense